solidity/test/relay/Relay.test.ts

```diff
@@ -559,6 +560,86 @@ describe("LightRelay", () => {
         ).to.be.revertedWith("Invalid target in new epoch")
       })
     })
+
+    context("after genesis (long chain)", () => {
+      const longGenesis = longHeaders.epochStart
+      const longGenesisBlock = 739872
+      const longRetargetBlock = 741888
+      const longHeaderStart = 741793
```

Comment:
Are they used anywhere?

Reply:
For convenience, can move into comments.